shell/common/common_TPC-H.sh

```diff
@@ -105,7 +138,7 @@ tpc-h_load-text(){
 }
 
 tpc-h_delete-text(){
-  #if [ ! "$BENCH_KEEP_FILES" == "1" ] && [ ! "$BENCH_LEAVE_SERVICES" "1"  ] ; then
+  #if [ ! "$BENCH_KEEP_FILES" == "1" ] && [ ! "$BENCH_LEAVE_SERVICES" == "1" ] ; then
```

Comment:
Not my code!

Reply:
So why is it in your merge request?

Reply:
Because that is code made by the Frankfurt guys, I tried to change as little code as possible from what the did and tested.

Reply:
If it's in your branch, it's your code 😉 unless you have a compelling reason to keep this line (and other random commented-out stuff) I suggest removing it before merge. Otherwise it just gets left there for ever => code rot.